docker/.env

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+export DOCKER_SOURCE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Comment:
a lot of quoting going on here, I'm guessing the inner quotes should be escaped?

Reply:
quotes allow ` dirname "${BASH_SOURCE[0]}` to expand to `.` where necessary. Without them it will resolve to `` which will fail because dirname expects a path  

```
echo $( dirname ${BASH_SOURCE[0]})
usage: dirname path
```

```
echo $( dirname "${BASH_SOURCE[0]}")
.
```